internal/bridge/client.go

```diff
@@ -724,10 +724,7 @@ func (c *Client) UpdateCluster(
 ) (*ClusterApiResource, error) {
 	result := &ClusterApiResource{}
 
-	clusterbyte, err := json.Marshal(clusterRequestPayload)
-	if err != nil {
-		return result, err
-	}
+	clusterbyte, _ := json.Marshal(clusterRequestPayload)
```

Comment:
interesting, i wonder why upstream decided to ignore the error

Reply:
https://github.com/percona/percona-postgresql-operator/pull/1094/commits/e57f1d674faec61e67f17731d94480b1fbe8be5e

Reply:
reverted this